server/src/main/kotlin/pishello/hello/api/AuthEndpoint.kt

```diff
@@ -20,10 +21,13 @@ class AuthEndpoint(val userPort: UserPort, val tokenPort: TokenPort) {
     }
 
     @PostMapping("/register", consumes = ["application/json"])
-    fun register(@RequestBody request: LoginRequest): ResponseEntity<Unit> {
+    fun register(@RequestBody request: RegisterRequest): ResponseEntity<Unit> {
         return if (!userPort.checkIfUserExists(request.name)) {
-            userPort.createNewUser(request.name, request.password)
-            ResponseEntity(HttpStatus.CREATED)
+            if (request.password == request.passwordRepeat) {
```

Comment:
Dlaczego sprawdzenie czy password == passwordRepeat odbywa się w serwerze? Takie sprawdzenie powinno odbyć się na froncie, jeśli hasła nie są identyczne wyświetlić komunikat użytkownikowi. A do serwera powinno przyjść już pod tym względem zwalidowane hasło

Reply:
Password checking moved to frontend